### ISSUE TITLE
Sort servers and channels

### ISSUE THREAD

**Issue by CarlosPL** [flackr/circ#361]
I understand CIRC is meant to be a simple IRC client, but some functions like being able to sort the list of channels and servers based of the user's criteria would be quite useful.

**Comment by flackr** [comment]
What custom criteria did you have in mind? Or did you want to manually order it?

**Comment by CarlosPL** [comment]
Yes, I wanted an option for manually sorting one's saved channels and
servers.

2016-02-16 17:00 GMT+01:00 Robert Flack <notifications@github.com>:
